src/utils/store-utils.ts

```diff
@@ -1,10 +1,12 @@
-import { devtools } from 'zustand/middleware';
-
 // TODO - add in Immer middleware
 // https://github.com/pmndrs/zustand#middleware
 // https://github.com/pmndrs/zustand/blob/main/tests/middlewareTypes.test.tsx
 
-export const devtoolsInNonProd = (process.env.NODE_ENV === 'production' ||
-process.env.NODE_ENV === 'test'
-    ? (fn: any) => fn
-    : devtools) as unknown as typeof devtools;
+export const devtoolsOptions = (name: string) => {
```

Comment:
No longer need the hack as https://github.com/pmndrs/zustand/pull/880 Fixed the issue.

Now we can just enable/disable the devtools as needed.